core-libs/setup/ssr/error-handling/server-error-response/propagate-server-error-response.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { CxServerErrorResponse } from './cx-server-error-response';
+
+/**
+ * Injection token used to provide a function that will propagate the server error response occurred during server side rendering to the CommonEngine wrapper.
+ *
+ * @private
```

Comment:
Why Private API?
What if some customer wants to have their own custom wrapper over CommonEngine - e.g. with a strategy to immediately propagate server error responses (instead of waiting until the rendering completes)?